.github/workflows/lint_commit_messages.yml

```diff
@@ -12,7 +12,10 @@ jobs:
       - name: Setup gitlint
         run: sudo apt-get install gitlint
       - name: Checkout
-        run: git clone https://github.com/SUSE/carrier.git && cd carrier && git checkout ${GITHUB_HEAD_REF}
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
oh nice! @thardeck I think this should solve the issue you had (which is why you manually cloned).